packages/mui-material/src/Button/Button.js

```diff
@@ -76,189 +60,218 @@ const ButtonRoot = styled(ButtonBase, {
       ownerState.fullWidth && styles.fullWidth,
     ];
   },
-})(
-  ({ theme, ownerState }) => {
-    const inheritContainedBackgroundColor =
-      theme.palette.mode === 'light' ? theme.palette.grey[300] : theme.palette.grey[800];
+})(({ theme }) => {
+  const inheritContainedBackgroundColor =
+    theme.palette.mode === 'light' ? theme.palette.grey[300] : theme.palette.grey[800];
 
-    const inheritContainedHoverBackgroundColor =
-      theme.palette.mode === 'light' ? theme.palette.grey.A100 : theme.palette.grey[700];
-
-    return {
-      ...theme.typography.button,
-      minWidth: 64,
-      padding: '6px 16px',
-      borderRadius: (theme.vars || theme).shape.borderRadius,
-      transition: theme.transitions.create(
-        ['background-color', 'box-shadow', 'border-color', 'color'],
-        {
-          duration: theme.transitions.duration.short,
-        },
-      ),
-      '&:hover': {
-        textDecoration: 'none',
-        backgroundColor: theme.vars
-          ? `rgba(${theme.vars.palette.text.primaryChannel} / ${theme.vars.palette.action.hoverOpacity})`
-          : alpha(theme.palette.text.primary, theme.palette.action.hoverOpacity),
-        // Reset on touch devices, it doesn't add specificity
-        '@media (hover: none)': {
-          backgroundColor: 'transparent',
-        },
```

Comment:
Aren't we missing this one for the `text` variant?

Reply:
Nice catch.

Reply:
Looking at it again, it's already covered.